### PR TITLE
Full support Italian - 1/2

### PR DESCRIPTION
**Pull request by rosastern**

### CODE REVIEW

File: grammar/fr/src/rules.rs [hunk 1]
```diff
@@ -314,12 +314,6 @@ pub fn rules_duration(b: &mut RuleSetBuilder<Dimension>) -> RustlingResult<()> {
              time_check!(),
              |duration, _, time| duration.value().after(time.value())
     );
-    b.rule_3("<duration> apres <time>",
```
Comment (Odile-deVismes) [inline grammar/fr/src/rules.rs]: Why did you remove this rule but let the rule <duration> avant <time>?

Comment (rosastern) [reply grammar/fr/src/rules.rs]: That was just a doublon, the rule was there twice
